### ISSUE TITLE
incorrect dependency resolution for libcpp-presets

### ISSUE THREAD

**Issue by kbrowder** [coursier/coursier#409]
The build file (actually simplified to the root cause)
```scala
libraryDependencies ++= Seq(
   "org.datavec" % "datavec-data-image" % "0.7.0"
)
```

On 1.0.0-M15 it results in this error:
```bat
coursier.ResolutionException: 1 not found
  https://repo1.maven.org/maven2/org/bytedeco/javacpp-presets/1.2/javacpp-presets-1.2-windows-x86_64.jar
[error] (*:update) coursier.ResolutionException: 1 not found
[error]   https://repo1.maven.org/maven2/org/bytedeco/javacpp-presets/1.2/javacpp-presets-1.2-windows-x86_64.jar
[error] Total time: 1 s, completed Dec 8, 2016 7:15:16 PM
```

This works without the plugin.

**Comment by kzys** [comment]
I can reproduce the issue with the CLI.

```
% ./cli/target/pack/bin/coursier fetch -t org.datavec:datavec-data-image:0.7.0
...
https://repo1.maven.org/maven2/org/bytedeco/javacpp-presets/ffmpeg/3.0.2-1.2/ff…
  100.0% [##########] 10.3 MiB (1.5 MiB / s)
  Error:
https://repo1.maven.org/maven2/org/bytedeco/javacpp-presets/1.2/javacpp-presets-1.2-macosx-x86_64.jar: NotFound(https://repo1.maven.org/maven2/org/bytedeco/javacpp-presets/1.2/javacpp-presets-1.2-macosx-x86_64.jar,Some(true))
./cli/target/pack/bin/coursier fetch -t org.datavec:datavec-data-image:0.7.0  16.73s user 2.94s system 70% cpu 28.037 total
%
```

**Comment by alexarchambault** [comment]
These jars seem not be pulled with the current master... hopefully for good reasons (I'm not sure about that though :-| ).

**Comment by teliatko** [comment]
I assume this is already fixed with M13-3. See https://github.com/alexarchambault/coursier/pull/290 and https://github.com/alexarchambault/coursier/issues/278. I've tried it with following:

```build.sbt
name := "cousier-409"

libraryDependencies ++= Seq(
   "org.datavec" % "datavec-data-image" % "0.7.0"
)
```

```project/plugins.sbt
addSbtPlugin("io.get-coursier" % "sbt-coursier" % "1.0.0-M15-3")
```

But using:

```
$ coursier fetch -t org.datavec:datavec-data-image:0.7.0
```

Results in;
```
https://repo1.maven.org/maven2/org/bytedeco/javacpp-presets/1.2/javacpp-presets-1.2-macosx-x86_64.jar: NotFound(https://repo1.maven.org/maven2/org/bytedeco/javacpp-presets/1.2/javacpp-presets-1.2-macosx-x86_64.jar,Some(true))
```

I've installed `coursier` via:
```
$ brew install --HEAD paulp/extras/coursier
```
Version what I have is:
```
$ pwd
/usr/local/Cellar/coursier/HEAD-e33ab63
```

I assume that `e33ab63` is git commit which is last commit in HEAD from today.

Hope this will help to locate the issue or workaround it.

**Comment by alexarchambault** [comment]
That seems to be fixed in `1.0.0-RC1`.
